Automattic/sensei
# Update lesson properties block to use with LM templates

Fixes #5666 

### Changes proposed in this Pull Request

* Updates the `sensei-lms/lesson-properties` block to make it workable with Learning Mode templates.

### Testing instructions


* Go to Sensei > Settings and select the Default template for Learning Mode.
* Edit the Lesson template and confirm that you see "Lesson Properties" block. It should look like `Length: 10 minutes|Difficulty: None`. The contents of this block and it's position in the template is arbitrary. It's there for the purpose of this PR demo.
* Confirm that you cannot edit the contents of this block.
* Edit any lesson in the editor and confirm the existing and new "Lesson Properties" blocks work as expected.
